src/Cassandra/Builder.cs

```diff
@@ -133,7 +137,8 @@ public Configuration GetConfiguration()
                 _queryOptions,
                 _addressTranslator,
                 _startupOptionsFactory,
-                _sessionFactoryBuilder);
+                _sessionFactoryBuilder,
+                driverMetricsProvider: _driverMetricsProvider);
```

Comment:
I try to fix this, but named parameter was required for the previous version of the code, because `driverMetricsProvider` was the last parameter with a default value. Now it is the first parameter with a default value)